widgets/Ariba-batchsource.json

```diff
@@ -68,15 +68,15 @@
           "label": "Ariba Client ID",
           "name": "clientId",
           "widget-attributes": {
-            "placeholder": "Client Id"
+            "placeholder": "Ariba Client ID"
           }
         },
         {
           "widget-type": "password",
           "label": "Ariba Client Secret",
           "name": "clientSecret",
           "widget-attributes": {
-            "placeholder": "Client Secret"
+            "placeholder": "Ariba Client Secret"
```

Comment:
We need to add Ariba before API Key tooltip too....

Reply:
Already added